### DISCUSSION TITLE
[Question] I would like to ask if postgresql is compatible with flet?

### DISCUSSION THREAD

**Discussion by Generoso84** [flet-dev/flet#1821] [Q&A]
### Question

I would like to ask if postgresql is compatible with flet? i want to make a ERP website, mobile app and window application and if possible for flet dev to use Postgresql do you have any tutorial that can provide. kindly enlighten me. thank you advance.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
You can use practically any Python library with Flet, but we don't have a tutorial for PGSQL yet.
